src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,129 @@
+package homework_2.random_chars_table;
+
+import lombok.Getter;
+import lombok.Setter;
+import java.util.*;
+
+@Getter
+@Setter
+public class RandomCharsTable {
+
+    public enum Strategy {
+        EVEN,
+        ODD
+    }
+    private int length;
+    private int width;
+    private Strategy strategy;
+    private int[][] table;
+
+    public void run() {
+        Scanner scanner = new Scanner(System.in);
+        String[] arr = scanner.nextLine().trim().split("\\s");
+
+        scanner.close();
+        try {
+            setParameters(arr);
+        } catch (NumberFormatException ex) {
+            System.out.println("Passed parameters should match the format [positive integer] [positive integer] [even|odd]!");
+            return;
+        } catch (IllegalArgumentException ex) {
+            System.out.println("Passed parameters should match the format [positive integer] [positive integer] [even|odd]!");
+            return;
+        }
+        initializeTable();
+        System.out.println(getStrForPrinting());
+        System.out.println(getStrForPrintingByStrategy(getStrategy()));
+    }
+
+    public void setParameters(String[] arr) {
+        if (arr.length != 3) {
+            throw new IllegalArgumentException();
+        }
+
+        for (int i = 0; i < 3; i++) {
+            if (i < 2) {
+                int j = Integer.parseInt(arr[i]);
+
+                if (j < 1) {
+                    throw new IllegalArgumentException();
+                }
+
+                if (i == 0) {
+                    setLength(j);
+                } else {
+                    setWidth(j);
+                }
+            } else {
+                String strategyStr = arr[i];
+
+                if (Strategy.EVEN.equals(Strategy.valueOf(strategyStr.toUpperCase())) || Strategy.ODD.equals(Strategy.valueOf(strategyStr.toUpperCase()))) {
```

Comment:
```suggestion
               if (strategyStr.toLowerCase(Locale.ROOT).equals("even") || strategyStr.toLowerCase(Locale.ROOT).equals("odd")) {
```
Условие в if получилась даже короче
Есть ли необходимость использовать enum в данном случае?